src/components/ModelSteps/Scan.jsx

```diff
@@ -49,36 +49,38 @@ const Scan = ({ currentStep }) => {
     setIsImportFileModalOpen(true)
```

Comment:
what is the difference with the `openFilePickerModal` method?

Reply:
I will rename `setIsImportFileModalOpen`, which is not clear enough.
`isImportFileModalOpen/setIsImportFileModalOpen` => manages the display of the `ActionMenu` to offer 2 choices to the user (import from his device or import from Drive)

`openFilePickerModal` allows to manage the display of the FilePicker component (to import from Drive)